src/app/core/export/query.service.ts

```diff
@@ -248,6 +249,25 @@ export class QueryService {
     }, 0);
   }
 
+  /**
+   * Returns the avg of the provided array as string.
+   * It can also handle integers in strings, e.g. "3".
+   * The average is only calculated if the value exists and is a valid number.
+   * @param data an integer array
+   * @param decimals the amount of decimals for the result, default 0
+   * @private
+   */
+  private avg(data: any[], decimals = 0): string {
+    const numbers = data
+      .map((d) => Number.parseInt(d))
+      .filter((i) => !Number.isNaN(i));
+    const result =
+      numbers.length === 0
+        ? 0
+        : numbers.reduce((i, sum) => sum + i, 0) / numbers.length;
```

Comment:
Should the avg be calculated over all values or just the valid/existing ones?

Reply:
I would ignore missing / invalid. (that might help if we have a mix of entity types or badly maintained data)